docs/network/network_binding_plugins.md

```diff
@@ -175,8 +175,9 @@ A possible alternative to a sidecar, to configure the domain vNIC.
 
 Supported types:
 
-- `tap` (from v1.1.1): The domain configuration is set to use an existing
+- `tap`: The domain configuration is set to use an existing
```

Comment:
Could you please elaborate why was the version removed?
Since we don't have a documentation per released version, it helps to understand when a feature was introduced.
Additionally, managedTap was added bellow with a version number so now its inconsistent.